src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -83,6 +83,7 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
 
   shouldSetTextContent(props : P) : boolean,
   resetTextContent(instance : I) : void,
+  areChildrenOffscreen(props : P) : boolean,
```

Comment:
I guess my question is whether there's another deprioritize case besides `hidden` where you want to deprioritize but not as much a single if it's offscreen.